flake.nix

```diff
@@ -43,6 +43,7 @@
 
             tools = {
               haskell-language-server = {};  # Must use haskell.nix, because the compiler version should match
+              ghcid = {};
```

Comment:
In general I want to reduce the number of tools we add here, as it can severely increase the time needed to do `nix develop`. Can't you get `ghcid` externally?